src/kaggle/api/kaggle_api_extended.py

```diff
@@ -332,6 +339,15 @@ class KaggleApi(KaggleApi):
     command_prefixes_allowing_anonymous_access = ('datasets download',
                                                   'datasets files')
 
+    # Attributes
+    competition_fields = ['ref', 'deadline', 'category', 'reward', 'teamCount','userHasEntered']

```

Comment:
It would be nice to share these with the other file, but I don't think it's critical to refactor right now.